CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ cmake_dependent_option(WITH_TOOLS "Compile with Tools" ON "MSVC" OFF)
 set(GAME_DIR "${PROJECT_SOURCE_DIR}/../darkmod" CACHE STRING "The directory where the game assets are stored")
 option(COPY_EXE "Copy executable to GAME_DIR after build" ON)
 set(THIRDPARTY_PLATFORM_OVERRIDE "" CACHE STRING "If you build conan dependencies yourself with custom platform_name, pass platform_name string here")
+option(ENABLE_TRACY "Enable tracy profiling support. Needs to be activated ingame by enabling the cvar tdm_enable_tracing." ON)
```

Comment:
I think cvar tdm_enableTracing or tdm_enable_tracing should be renamed to com_enableTracing.

tdm_XXX cvar name was the convention back when the engine was closed source. As the result, it is almost exclusively about TDM game code. Today we try to add cvar like ID did, and tracing thing is about engine for sure.